.github/workflows/pullrequests.yml

```diff
@@ -115,6 +115,14 @@ jobs:
           TMP_CLOUDFLARE_API_TOKEN: ${{ secrets.CLOUDFLARE_API_TOKEN }}
           TMP_CLOUDFLARE_ACCOUNT_ID: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
           NODE_OPTIONS: "--max_old_space_size=8192"
+          WRNALGER_LOG_PATH: ${{ runner.temp }}/wrangler-debug-logs/
```

Comment:
```suggestion
          WRANGLER_LOG_PATH: ${{ runner.temp }}/wrangler-debug-logs/
```

Reply:
argh good catch!

Reply:
looks like the typo survived the most recent commits